src/it/MSOURCES-121/pom.xml

```diff
@@ -29,9 +29,10 @@
 
   <name>Test for multiple attachments of files</name>
 	<description>This build should fail based on the duplicate
-   execution with  the same configuration. This will errornously
-   add the classifier/file twice times.
+   execution with  the same configuration. This will erroneously
```

Comment:
One small detail: multiple spaces.

Reply:
ok, while at it, let's fix 2 typos for the price of 1 :)